mantra-events/templates/Cargo.toml.gotmpl

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "my_project"
```

Comment:
Templated name, please fix for missing ones too (if any, in this PR it seems ok)
```suggestion
name = "{{ .Name }}"
```